src/pages/Analysis/index.js

```diff
@@ -0,0 +1,181 @@
+                </Grid>
+              </GridItem>
+              <GridItem span={5} style={{ borderLeft: '1px solid #b3b2b2' }}>
+                <div className={styles.paddingBig}>
+                  <TextContent>
+                    <Text component={TextVariants.h3}>example.eng.bos.redhat.com</Text>
+                    <Text component={TextVariants.h4}>Field</Text>
+                    <a>run.controller</a>
+                    <Text component={TextVariants.h4}>Value</Text>
+                    <a>scale-ci-example</a>
+                    <Text component={TextVariants.h4}>Count</Text>
+                    <span>54(3.81%)</span>
+                  </TextContent>
+                </div>
+              </GridItem>
+            </Grid>
+          </Card>
+          <Card className={styles.borderedCard}>
+            <div className={styles.paddingBig}>
+              <TextContent>
+                <span>
+                  <Text component={TextVariants.h3} style={{ display: 'inline' }}>
+                    Legend
+                  </Text>
+                </span>
+                <span className={styles.colorBoxInline} style={{ backgroundColor: 'blue' }} />
+                <span className={styles.legendLabel}>example.controller.com</span>
+                <span className={styles.colorBoxInline} style={{ backgroundColor: 'green' }} />
+                <span className={styles.legendLabel}>example.controller.com</span>
+                <span className={styles.colorBoxInline} style={{ backgroundColor: 'red' }} />
+                <span className={styles.legendLabel}>example.controller.com</span>
+                <span className={styles.colorBoxInline} style={{ backgroundColor: 'violet' }} />
+                <span className={styles.legendLabel}>example.controller.com</span>
+                <span className={styles.colorBoxInline} style={{ backgroundColor: '#f0ab00' }} />
+                <span className={styles.legendLabel}>example.controller.com</span>
+              </TextContent>
+            </div>
+          </Card>
+        </GridItem>
+      </Grid>
+      <Grid className={styles.paddingSmall}>
+        <GridItem span={12}>
+          <Card className={styles.borderedCard}>
+            <div className={styles.paddingBig}>
+              <TextContent>
+                <Text component={TextVariants.h2}>
+                  {' '}
+                  Runs over time
+                  <Button variant="link" icon={<ExternalLinkAltIcon />} style={{ float: 'right' }}>
+                    View in Kibana
+                  </Button>
+                </Text>
+              </TextContent>
+            </div>
+          </Card>
+          <Card className={styles.borderedCard}>
+            <Grid>
+              <GridItem span={8}>
+                <TextContent className={styles.paddingMedium}>
+                  <Text component={TextVariants.h4} style={{ color: 'grey' }}>
+                    {' '}
+                    @metadata.satellite.EC2: filters
```

Comment:
Again, this looks "hardcoded" ... `@metadata.satellite` is a field in the document, but `EC2` is a particular satellite designation, the Elasticsearch "keyword" value of `@metadata.satellite`. It might be useful to filter on data from a particular satellite, for example, but the "EC2" name is specific to our configuration.